scripts/deploy.sh

```diff
@@ -37,13 +38,19 @@ DEPLOYMENT_NAME=${DEPLOYMENT_NAME:-"kubeflow"}
 
 KUBEFLOW_KS_DIR=${KUBEFLOW_KS_DIR:-"`pwd`/${DEPLOYMENT_NAME}_ks_app"}
 
+if [[ ! -z "$KUBEFLOW_DOCKER_REGISTRY" ]]; then
```

Comment:
I think this should be a separate script.

I think the flow we want is

1. User creates an APP with default configs
2. User performs any necessary customizations
3. User deploys